frontend/package.json

```diff
@@ -2,31 +2,25 @@
   "name": "bowtie",
   "version": "1.0.0",
   "description": "The UI displays the outcomes generated by Bowtie, which serves as a meta-validator for the JSON Schema specification",
-  "homepage": "/bowtie",
   "scripts": {
```

Comment:
Don't we need to add the `homepage` or is it something related to _**vite**_?

Reply:
Setting homepage was a thing for `create-react-app`, it indicated on which path the app was hosted. Now this part of config was moved to `vite.config.js` file